pkg/logql/metrics.go

```diff
@@ -173,6 +173,26 @@ func RecordRangeAndInstantQueryMetrics(
 		"cache_result_hit", resultCache.EntriesFound,
 		"cache_result_download_time", resultCache.CacheDownloadTime(),
 		"cache_result_query_length_served", resultCache.CacheQueryLengthServed(),
+		// The total of chunk reference fetched from index.
+		"ingester_chunk_refs", stats.Ingester.Store.GetTotalChunksRef(),
+		// Total number of chunks fetched.
+		"ingester_chunk_downloaded", stats.Ingester.Store.GetTotalChunksDownloaded(),
+		// Time spent fetching chunks in nanoseconds.
+		"ingester_chunk_fetch_time", stats.Ingester.Store.ChunksDownloadDuration(),
```

Comment:
we may want to take these back out? ingesters don't fetch chunks so I think this will always be zero values?